.travis.yml

```diff
@@ -3,13 +3,19 @@ os:
 
 language: ruby
 
+branches:
```

Comment:
is this accidental?

Reply:
nah, just copying from all the other CocoaPods CI configs

Reply:
I would rather us not do that, because getting CI to run before opening a PR is nice, but :woman_shrugging:

Reply:
But then you have CI running twice for each PR commit.... I've always just opened WIP PRs

Reply:
Fair :woman_shrugging: - :shipit: when CI is :green_apple:

Reply:
I'm waiting on travis for that :/